core/src/main/java/org/apache/iceberg/GenericManifestFile.java

```diff
@@ -404,6 +404,7 @@ public int hashCode() {
   @Override
   public String toString() {
     return MoreObjects.toStringHelper(this)
```

Comment:
I think we want this change even if we don't include the print method

Reply:
Done, split it out in #4185